finetune/nn/target_blocks.py

```diff
@@ -192,12 +191,6 @@ def classifier(hidden, targets, n_targets, config, train=False, reuse=None, **kw
                 clf_losses, targets, kwargs.get("class_weights")
             )
 
-            # From Unsupervised Data Augmentation for Consistency Training, Xie et al. 2019
-            if config.tsa_schedule:
```

Comment:
What were the results of the benchmarks here, out of curiosity? No substantial lift?